model_learning.py

```diff
@@ -177,11 +177,6 @@ def make_state_sequences(episode_subsequences, state_traces):
             continue
         counter += 1
 
-        if '10.0.254' not in attack:  # TODO: move all these checks to the `_group_alerts_per_team` function
-            continue
-        if '147.75' in attack or '69.172' in attack:
```

Comment:
I am not sure whether you need this check. For the CCDC dataset, I have always commented it out. Such IP parts are only present in CPTC-2018:
![image](https://github.com/tudelft-cda-lab/SAGE/assets/45541174/dae740a3-450e-450c-be12-39b90c841f76)

Running CPTC with this check commented out does not seem to change the AGs. Do you still want to keep this check? What is the idea of it?

Reply:
I am referring to the line `if '147.75' in attack or '69.172' in attack:`

Reply:
UPDATE: as you can also see in the description, moving this check to the check for `10.0.254` is redundant: the corresponding IPs will be already filtered by the `10.0.254` check, as you can see in the image below (this are the "missing" traces, but with the corresponding attacker-victim pair; the same as in the description):
![image](https://github.com/tudelft-cda-lab/SAGE/assets/45541174/50ebaa08-733c-4654-b2e0-f68bd8e4b6ef)